### PR TITLE
use uncompressed internal nodes for verkle

### PR DESCRIPTION
**Pull request by tanishqjasoria**
Fixes Closes Resolves #

_Please choose one of the keywords above to refer to the issue this PR solves followed by the issue number (e.g. Fixes #000). If no issue number, remove the line. Also, remove everything marked optional that is not applicable. Remove this note after reading._

## Changes

- _List the changes_

#### What types of changes does your code introduce?

- [ ] Bugfix (a non-breaking change that fixes an issue)
- [ ] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a change that causes existing functionality not to work as expected)
- [ ] Optimization
- [ ] Refactoring
- [ ] Documentation update
- [ ] Build-related changes
- [ ] Other: _Description_

#### Requires testing

- [ ] Yes
- [ ] No

#### If yes, did you write tests?

- [ ] Yes
- [ ] No

#### Notes on testing

_Optional. Remove if not applicable._

#### Requires documentation update

- [ ] Yes
- [ ] No

_If yes, link the PR to the docs update or the issue with the details labeled `docs`. Remove if not applicable._

#### Requires explanation in Release Notes

- [ ] Yes
- [ ] No

_If yes, fill in the details here. Remove if not applicable._

## Remarks

_Optional. Remove if not applicable._

### CODE REVIEW

File: src/Nethermind/Nethermind.Verkle.Tree.Test/NodeSerializerTests.cs [hunk 1]
```diff
@@ -45,13 +44,13 @@ public void TestFullStemNode()
         while (true)
         {
             _rand.NextBytes(c1);
-            c1Point = Banderwagon.FromBytes(c1);
+            c1Point = Banderwagon.FromBytesUncompressedUnchecked(c1);
```
Comment (jjeangal) [inline src/Nethermind/Nethermind.Verkle.Tree.Test/NodeSerializerTests.cs:47]: Add is BigEndian: false

File: src/Nethermind/Nethermind.Verkle.Tree.Test/NodeSerializerTests.cs [hunk 2]
```diff
@@ -45,13 +44,13 @@ public void TestFullStemNode()
...
             if (c1Point.HasValue) break;
         }
         while (true)
         {
             _rand.NextBytes(c2);
-            c2Point = Banderwagon.FromBytes(c2);
+            c2Point = Banderwagon.FromBytesUncompressedUnchecked(c2);
```
Comment (jjeangal) [inline src/Nethermind/Nethermind.Verkle.Tree.Test/NodeSerializerTests.cs:53]: Add is BigEndian: false

File: src/Nethermind/Nethermind.Verkle.Tree.Test/NodeSerializerTests.cs [hunk 3]
```diff
@@ -74,14 +73,14 @@ public void TestC1NullStemNode()
         while (true)
         {
             _rand.NextBytes(commitment);
-            internalCommitment = Banderwagon.FromBytes(commitment);
+            internalCommitment = Banderwagon.FromBytesUncompressedUnchecked(commitment);
```
Comment (jjeangal) [inline src/Nethermind/Nethermind.Verkle.Tree.Test/NodeSerializerTests.cs:76]: Add isBigEndian: false

File: src/Nethermind/Nethermind.Verkle.Tree.Test/NodeSerializerTests.cs [hunk 4]
```diff
@@ -74,14 +73,14 @@ public void TestC1NullStemNode()
...
             if (internalCommitment.HasValue) break;
         }
 
         while (true)
         {
             _rand.NextBytes(c2);
-            c2Point = Banderwagon.FromBytes(c2);
+            c2Point = Banderwagon.FromBytesUncompressedUnchecked(c2);
```
Comment (jjeangal) [inline src/Nethermind/Nethermind.Verkle.Tree.Test/NodeSerializerTests.cs:83]: Add is BigEndian: false

File: src/Nethermind/Nethermind.Verkle.Tree.Test/NodeSerializerTests.cs [hunk 5]
```diff
@@ -94,22 +93,22 @@ public void TestC1NullStemNode()
...
+        byte[] c1 = new byte[64];
         byte[] stem = new byte[31];
 
         Banderwagon? c1Point;
         Banderwagon? internalCommitment;
         while (true)
         {
             _rand.NextBytes(commitment);
-            internalCommitment = Banderwagon.FromBytes(commitment);
+            internalCommitment = Banderwagon.FromBytesUncompressedUnchecked(commitment);
```
Comment (jjeangal) [inline src/Nethermind/Nethermind.Verkle.Tree.Test/NodeSerializerTests.cs:105]: Add isBigEndian: false

File: src/Nethermind/Nethermind.Verkle.Tree.Test/NodeSerializerTests.cs [hunk 6]
```diff
@@ -94,22 +93,22 @@ public void TestC1NullStemNode()
...
             if (internalCommitment.HasValue) break;
         }
         while (true)
         {
             _rand.NextBytes(c1);
-            c1Point = Banderwagon.FromBytes(c1);
+            c1Point = Banderwagon.FromBytesUncompressedUnchecked(c1);
```
Comment (jjeangal) [inline src/Nethermind/Nethermind.Verkle.Tree.Test/NodeSerializerTests.cs:111]: Add isBigEndian: false

File: src/Nethermind/Nethermind.Verkle.Tree.Test/NodeSerializerTests.cs [hunk 7]
```diff
@@ -122,14 +121,14 @@ public void TestC2NullStemNode()
...
+        byte[] commitment = new byte[64];
         byte[] stem = new byte[31];
 
         Banderwagon? internalCommitment;
         while (true)
         {
             _rand.NextBytes(commitment);
-            internalCommitment = Banderwagon.FromBytes(commitment);
+            internalCommitment = Banderwagon.FromBytesUncompressedUnchecked(commitment);
             if (internalCommitment.HasValue) break;
```
Comment (jjeangal) [inline src/Nethermind/Nethermind.Verkle.Tree.Test/NodeSerializerTests.cs:132]: Add isBigEndian: false

**Review by jjeangal — COMMENTED** [review]
I believe there is the attribute isBigEndian = false missing at a couple of places

**Comment by tanishqjasoria** [comment]
> I believe there is the attribute isBigEndian = false missing at a couple of places

it kind of does not matter as we are just getting random values for tests, but I changed it for consitency.

**Pull request merged**